AudioManagerUtilities/include/CAmSocketHandler.h

```diff
@@ -291,7 +291,9 @@ class CAmSocketHandler
     VectorListPoll_t mListPoll; //!<list that holds all information for the ppoll
     sh_identifier_s mSetTimerKeys; //!A set of all used timer keys
     std::list<sh_timer_s> mListTimer; //!<list of all timers
+#ifndef WITH_TIMERFD
```

Comment:
Note sure if we discussed #ifdefs before?  Is the plan to decide which approach to go for and eventually remove the #ifdefs?  It's not that nice to have them - maybe it could be changed to a runtime configuration instead.

--> Question only

Reply:
In future I would suggest to remove the parallel hosting of old and new timer implementation.
I would suggest to stick with timerfd implementation only.